dndwiki/races/race-concubus.xml

```diff
@@ -0,0 +1,98 @@
+        Even to the most astute observers, your ruse is usually indiscernible. If you rouse suspicion, or if a wary creature suspects something is amiss, you have advantage on any Charisma (Deception) check you make to avoid detection of your true nature.<br/>
+        <span class="feature">Silver Tongue.</span>You have proficiency with the Persuasion skill. If you spend 1 minute talking to someone who can understand what you say, you can make a Charisma (Persuasion) check contested by the creature's Wisdom (Insight) check. If you or your companions are fighting the creature, your check automatically fails. If your check succeeds, the target is magically charmed by you as long as it remains within 60 feet of you and for 1 minute thereafter.<br/>
+        <span class="feature">Wings.</span>You have a pair of wings that give you a flying speed of 30 feet. You cannot use this fly speed if you are wearing medium or heavy armor or if your current form granted by your shapechanger trait does not have wings.<br/>
+        <span class="feature">Languages.</span>You can speak, read, and write Common and your choice of Abyssal or Infernal.<br/>
+      </p>
+    </description>
+    <sheet display="false" />
+    <setters>
+      <set name="height" modifier="2d10">4'9"</set>
+      <set name="weight" modifier="2d4">110 lb.</set>
+      <set name="sourceUrl">https://www.dandwiki.com/wiki/Incubus/Succubus_(5e_Race)</set>
+    </setters>
+    <rules>      
+      <stat name="dexterity" value="1"/>
+      <stat name="charisma" value="1"/>
+      <stat name="innate speed" value="30" bonus="base"/>
+      <stat name="innate speed:fly" value="30" bonus="base" equipped="![armor:medium],![armor:heavy]" />
+      <grant type="Size" id="ID_SIZE_MEDIUM" />
+      <grant type="Vision" id="ID_VISION_DARKVISION" />
+      <grant type="Language" id="ID_LANGUAGE_COMMON" />
+      <select type="Language" supports="ID_LANGUAGE_INFERNAL|ID_LANGUAGE_ABYSSAL" />
+      <grant type="Racial Trait" id="ID_RACIAL_TRAIT_FIENDISH_NATURE" />
+      <grant type="Racial Trait" id="ID_RACIAL_TRAIT_SHAPECHANGER" />
+      <grant type="Racial Trait" id="ID_RACIAL_TRAIT_SILVER_TONGUE" />
+      <grant type="Racial Trait" id="ID_RACIAL_TRAIT_WINGS" />
```

Comment:
Update IDs to include `_WIKI_` and the name of the race `_SUCCUBUS_`
e.g. `ID_RACIAL_TRAIT_WINGS` =>`ID_WIKI_RACIAL_TRAIT_SUCCUBUS_WINGS`